subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -0,0 +1,212 @@
+# Bluetooth Audio - Call control configuration options
+#
+# Copyright (c) 2020 Bose Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+#####################  Telephone Bearer Service #####################
+
+config BT_TBS
+	bool "Telephone Bearer Service Support"
+	select BT_CCID
+	help
+	  This option enables support for Telephone Bearer Service.
+
+if BT_TBS
+
+# TODO: BT_GTBS is mandatory if you support the call control server role.
+# Need to enforce this.
+config BT_GTBS
+	bool "Generic Telephone Bearer Service Support"
+	default y
+	help
+	  This option enables support for Generic Telephone Bearer Service.
+
+config BT_TBS_PROVIDER_NAME
+	string "Telephone Bearer Service Provider Name"
+	default "Unknown"
+	help
+	  Sets the name of the service provider for the bearer.
+
+config BT_TBS_UCI
+	string "Telephone Bearer Service Uniform Caller Identifier (UCI)"
+	default "un000"
+	help
+	  Sets the UCI of the bearer. See
+		https://www.bluetooth.com/specifications/assigned-numbers/uniform-caller-identifiers/
+		for a table of valid UCIs.
+
+config BT_TBS_TECHNOLOGY
+	int "Telephone Bearer Service Technology"
+	range 1 10
+	help
+	  Sets the technology used for the bearer, e.g. GSM, LTE and 5G.
+	  1 : 3G
+	  2 : 4G
+	  3 : LTE
+	  4 : Wi-Fi
+	  5 : 5G
+	  6 : GSM
+	  7 : CDMA
+	  8 : 2G
+	  9 : WCDMA
+	  10: IP
+
+config BT_TBS_URI_SCHEMES_LIST
+	string "Telephone Bearer Service URI schemes Supported List"
+	default "tel,skype"
+	help
+	  Sets a list of URI schemes that are supported by the bearer,
+	  e.g. "tel" or "skype".
+	  Multiple values shall be comma (,) separated, e.g. "tel,skype".
+
+config BT_TBS_SIGNAL_STRENGTH_INTERVAL
+	int "Telephone Bearer Service Signal Strength Reporting Interval"
+	default 0
+	range 0 255
+	help
+	  Sets the interval of reporting the signal strength in seconds.
+		If the value is 0, the signal will not be reported.
```

Comment:
As far as I remember the signal strength interval is configurable by the client, so why is there a need to have it in Kconfig?

Reply:
This will be the initial value. Once we start registering the TBS dynamically this will be removed.